packages/seedless-onboarding-controller/package.json

```diff
@@ -51,6 +51,9 @@
     "@metamask/base-controller": "^8.0.1",
     "@metamask/toprf-secure-backup": "^0.4.0",
     "@metamask/utils": "^11.4.2",
+    "@noble/ciphers": "^1.3.0",
+    "@noble/curves": "^1.9.2",
+    "@noble/hashes": "^1.8.0",
```

Comment:
Nit: instead of using `no-changelog` could we add a changelog entries for these bumps and that they are being now used as dependencies

Reply:
done